pkg/runtime/ast/ast.go

```diff
@@ -1757,3 +1759,44 @@ func (cc *convCtx) convKill(stmt *ast.KillStmt) Statement {
 		ConnectionID: stmt.ConnectionID,
 	}
 }
+
+// Convert mysql optimizer hints
+// Include https://dev.mysql.com/doc/refman/8.0/en/optimizer-hints.html#optimizer-hints-index-level
+func (cc *convCtx) convTableHint(stmt []*ast.TableOptimizerHint) *HintNode {
+	// ignore hints filter
+	var isIgnore = func(hintName string) bool {
+		ignoreMap := map[string]string{
+			"TIDB_HJ": "tidb hj hints",
+		}
```

Comment:
Use a global variable instead.

Reply:
Got it, I will optimize this in next commit